src/colors/colors.css

```diff
@@ -28,4 +28,10 @@
   --omrs-color-brand-gold: #eea616;
   --omrs-color-brand-teal: #009384;
   --omrs-color-brand-black: #231f20;
+
+  --omrs-bg-immunization-sequence-odd-row: #c4d7e6;
```

Comment:
The styleguide is for the whole app and does not know about sub-regions.

So either call these `--omrs-bg-table-odd-row` etc. or bring these values to the domain where they belong.

Reply:
Hello @FlorianRappl,
Thanks for the comment. I thought `esm-styleguide` is the place where we are defining the color variables for the whole applications. If that's not the case, we could abandon this PR and I could provide them from widgets itself.

Reply:
For the "whole" application, but not for domain-specific parts.

Just think about it like that: Somebody wants to add a new microfrontend for "foo". Where should the style go in? If this style would need to be added in here, then the whole microfrontend approach does not bring you anything. If this repository *needs* to change when something new should be added then the approach is not sound.

In other words: Just because I want a new stereo in my car does not mean I need to exchange the engine.

=> So if this is a general improvement then name it properly (e.g., like suggested), otherwise bring it to the specific domain.

Reply:
+1 @FlorianRappl , all of these should be defined in the Immunization package itself.

But please take a look at [the styleguide](https://styleguide.openmrs.org/?path=/story/openmrs-styleguide--colors) and try to use colors from there if they are reasonably close. This will help the styling be more coherent across the application. For example, consider using the `--omrs-color-bg-` colors in place of your custom colors, if you're able to get something close enough to the desired effect by doing so.

Reply:
Thanks @brandones @FlorianRappl,
I got the point around that these should be defined in the immunization widget itself. I will move them over there and abandon this PR.

@brandones I will also look into the colors suggested here. However I would be defining these as variables and not as hardcoded colors so they could be overridden for refapp.

Thanks

Reply:
@brandones @FlorianRappl 
I have moved these colors to Immunization specific css classes and tried to use the colors from the styleguide as much as possible.
I will abandon this PR here.